pkg/manager/services.go

```diff
@@ -162,13 +162,32 @@ func (sm *Manager) addService(svc *v1.Service) error {
 	// Check if egress is enabled on the service, if so we'll need to configure some rules
 	if svc.Annotations[egress] == "true" && len(serviceIPs) > 0 {
 		log.Debugf("Enabling egress for the service [%s]", svc.Name)
-		if svc.Annotations[activeEndpoint] != "" {
-			// We will need to modify the iptables rules
-			err = sm.iptablesCheck()
-			if err != nil {
-				log.Errorf("Error configuring egress for loadbalancer [%s]", err)
+		// We will need to modify the iptables rules
+		err = sm.iptablesCheck()
+		if err != nil {
+			log.Errorf("Error configuring egress for loadbalancer [%s]", err)
+		}
+		var podIP string
+		errList := []error{}
+
+		// Should egress be IPv6
+		if svc.Annotations[egressIPv6] == "true" {
```

Comment:
With this new annotation kube-vip will check for IPv6 address and if there is no `egressIPv6` annotation it will process service's egress as IPv4.
Wouldn't that effectively disable any support for dualstack services?

Reply:
This change only affects anything `egress` related, the services/load balancers are all created as usual but if the correct annotations exist then egress rules are created too. The dual stack service test is all passing in the CI/CD.